src/Button.js

```diff
@@ -1,18 +1,40 @@
-import React from "react";
-import "./Style.css";
-import constant from "./constants";
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './calculatorStyle.css';
 
-let colors = constant.colors;
-const Button = props => (
-  <button
-    className="Button"
-    style={{
-      width: props.value === "0" ? "120px" : "60px",
-      backgroundColor: colors[props.index]
-    }}
-  >
-    {props.value}
-  </button>
-);
+class Button extends React.Component {
+    render() {
+        const {
+            value = 'Button',
+            color = 'light gray',
+            width = 'narrow',
+            updateState,
+        } = {
+            ...this.props,
+        };
+        const classes = classNames({
+            'button-style': true,
+            lg: color === 'light gray',
```

Comment:
`sm`, `md`, `lg` are well-known names for dimensions and they are ok to be used like this. But when `lg` means 'light grey' things get confusing. Therefore, please use `light-grey`, `dark-grey`, `orange` and so on, and `narrow` and `wide`, because nobody will know what those 2 or 3 letters mean.